client/app/actions/message_action_creator.coffee

```diff
@@ -1,7 +1,7 @@
 {ActionTypes}   = require '../constants/app_constants'
 XHRUtils        = require '../libs/xhr'
 
-MessageGetters  = require '../puregetters/message'
+MessageGetters  = require '../puregetters/messages'
```

Comment:
Could you add into this PR's description le list of the functional scenari you fixed (such as this one) plz?